sources/packages/web/src/composables/useRules.ts

```diff
@@ -38,18 +38,70 @@ export function useRules() {
     return `${fieldName} is required.`;
   };
 
+  /**
+   * Check if a value is truthy.
+   * All values are truthy except false, 0, -0, 0n,
+   * "", null, undefined, and NaN.
+   * @param value value to be checked.
+   * @param fieldName friendly field name to be added
+   * to the validation message.
+   * @returns true if the value is truthy,
+   * otherwise a validation message.
+   */
+  const truthyRule = (value: string, fieldName: string) => {
```

Comment:
We have an existing rule already

![image](https://user-images.githubusercontent.com/54600590/218622948-4d8ee58e-3ec3-450a-bfca-e88e706f6cd5.png)

Reply:
Adpted the current one to accept also `number` as the value that is the type that came from the input defined as a number and added some comments.

Reply:
thanks for the modification to current method